src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -32,24 +32,32 @@ using namespace ov;
 using namespace ov::frontend::tensorflow;
 
 namespace {
-std::vector<std::string> get_unconverted_types_from_model(const std::shared_ptr<Model>& model) {
-    std::vector<std::string> unconverted_ops_types;
+
+void get_unsupported_operations_and_failures(const std::shared_ptr<Model>& model,
+                                             std::vector<std::string>& unsupported_operations,
+                                             std::unordered_map<std::string, std::string>& failures) {
+    unsupported_operations.clear();
```

Comment:
When the function is called recursively (in case when `MultiSubGraphOp` is discovered below in the code) this will erase all accumulated content with unsupported operations. This makes output incomplete: not all the unsupported operations are reported.

Reply:
Fixed. Great catch!